daemons/attrd/attrd_cib.c

```diff
@@ -45,21 +45,16 @@ attrd_cib_destroy_cb(gpointer user_data)
 }
 
 static void
-attrd_cib_replaced_cb(const char *event, xmlNode *msg)
+attrd_cib_updated_cb(const char *event, xmlNode *msg)
 {
     const char *client_id = crm_element_value(msg, F_CIB_CLIENTID);
-    int change_section = cib_change_section_nodes
-                         |cib_change_section_status
-                         |cib_change_section_alerts;
 
-    if (attrd_shutting_down(true)) {
+    if (attrd_shutting_down(false)) {
+        // @TODO Pass true? attrd_cib_replaced_cb() passed true.
```

Comment:
Definitely true (see a1a9c54cfc). If we wanted to preserve existing behavior, we could check it with false before checking alerts and then call it again with true before checking nodes. However I think it's reasonable to ignore any changes to alerts if we're intending to shut down.